CDP4JsonSerializer/CDP4JsonSerializer.csproj

```diff
@@ -20,37 +20,31 @@
     <PackageTags>CDP COMET ECSS-E-TM-10-25 JSON</PackageTags>
     <PackageLicenseExpression>LGPL-3.0-only</PackageLicenseExpression>
     <PackageReleaseNotes>
-        [BUMP] To CDP4Common 27.4.0
+      [Refactor] Newtonsoft to System.Text.Json
     </PackageReleaseNotes>
-    <PackageReadmeFile>README.md</PackageReadmeFile>
+      <PackageReadmeFile>README.md</PackageReadmeFile>
+    <LangVersion>11.0</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.3" PrivateAssets="All" />
+    <PackageReference Include="NLog" Version="5.3.4" />
```

Comment:
this should not be necessary since this is a transitive dependency from CDP4Common, please remove